Index: src/main/java/nl/pekict/year2021/Day12.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/nl/pekict/year2021/Day12.java b/src/main/java/nl/pekict/year2021/Day12.java
new file mode 100644
--- /dev/null	(date 1639390535374)
+++ b/src/main/java/nl/pekict/year2021/Day12.java	(date 1639390535374)
@@ -0,0 +1,31 @@
+package nl.pekict.year2021;
+
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class Day12 {
+
+    public static void main(String[] args) {
+        solution();
+    }
+
+    public static void solution() {
+        // Part 1
+        long startTime = System.currentTimeMillis();
+        int[][] grid = new int[3][5];
+
+        try (Scanner scanner = new Scanner(Paths.get("day12.txt"))) {
+            while (scanner.hasNextLine()) {
+                String[] pieces = scanner.nextLine().split("-");
+            }
+        } catch (Exception e) {
+            System.out.println("Error: " + e.getMessage());
+        }
+        long endTime = System.currentTimeMillis();
+        long totalTime = endTime - startTime;
+
+        System.out.println("Antwoord 1: ");
+        System.out.println("Antwoord 2: ");
+        System.out.println("Tijd: " + totalTime + "ms");
+    }
+}
